File: scala/src/main/scala/com/vdzon/administratie/checkandfix/model/CheckAndFixRegel.scala
package com.vdzon.administratie.checkandfix.model

import java.time.LocalDate

import com.vdzon.administratie.dto.AfschriftDto

case class CheckAndFixRegel (val rubriceerAction: FixAction = null,
  val omschrijving: String = null,
  val data: String = null,
  val boekingUuid: String = null,
  val checkType: CheckType = null,
  val afschrift: AfschriftDto = null,
  val date: LocalDate = null);

File: checkandfix/api/src/main/java/com/vdzon/administratie/checkandfix/CheckAndFixRegel.kt
package com.vdzon.administratie.checkandfix

import com.vdzon.administratie.checkandfix.model.CheckType
import com.vdzon.administratie.checkandfix.model.FixAction
import com.vdzon.administratie.dto.AfschriftDto
import java.time.LocalDate

class CheckAndFixRegel(val rubriceerAction: FixAction? = null,
                       val omschrijving: String? = null,
                       val data: String? = null,
                       val boekingUuid: String? = null,
                       val checkType: CheckType? = null,
                       val afschrift: AfschriftDto? = null,
                       val date: LocalDate? = null);
